### DISCUSSION TITLE
Transfer todos/tasks to other open todo.txt files

### DISCUSSION THREAD

**Discussion by mshuflin** [ransome1/sleek#696] [Ideas]
When reviewing my todo list, sometimes I would like to move a todo from my todo.txt to another list (e.g. somedayMaybe.txt). Currently, to do that, I have to click the todo, select the text, copy, mark the todo complete, and then move to the other tab, create a new todo and paste.

If we could right click and choose to "transfer" (essentially cut and paste) a todo from any open file to another, that would significantly speed up the review process.

**Comment by mfvianna** [comment]
Dragging the todo item to the other file TAB would be even nicer
